docker/package/wizard_structure.py

```diff
@@ -363,6 +363,18 @@ def search_json_with_default(json_filepath, field, default):
         return json_dict.pop(field, default)
 
 
+def replace_in_service_config(config_filepath, field, value):
+    with open(config_filepath, "r") as f:
+        config_contents = f.read()
+
+    old = re.search(f"{field}=.*", config_contents)
+    if old is None:
+        return None
+    else:
+        new = f"{field}={value}"
+        proc_call(f"sudo sed -i 's/{old.group(0)}/{new}' {config_filepath}")
```

Comment:
I got the following `sed` error, when wizard attempted to update config file, seems like `/` is missing a the end.
```
You can check a blockchain explorer (e.g. https://tzkt.io/ or https://tzstats.com/)
to see the baker status and baking rights of your account.
Would you like to request to end the Liquidity Baking subsidy?
1. pass (default)              Abstain from voting
2. off                         Request to end the subsidy
3. on                          Request to continue or restart the subsidy
> 2
sed: -e expression #1, char 72: unterminated `s' command
Error in Tezos Setup Wizard, exiting.
The error has been logged to /home/roman/tezos_setup_wizard.log
```
```suggestion
        proc_call(f"sudo sed -i 's/{old.group(0)}/{new}/' {config_filepath}")
```